org.adempiere.base/src/org/compiere/util/DB.java

```diff
@@ -2506,6 +2506,43 @@ else if (metadata.storesLowerCaseIdentifiers())
     	return rowsArray;
 	}
 
+	/**
+	 * Get Boolean Value from SQL
+	 * 
+	 * @param  trxName     trx
+	 * @param  sql         sql
+	 * @param  params      array of parameters
+	 * @return             first boolean value or false
+	 * @throws DBException if there is any SQLException
+	 */
+	public static boolean getSQLValueBooleanEx(String trxName, String sql, Object... params)

```

Comment:
Oracle doesn't have a Boolean data type - it could be implemented wrapping another type, but as that's more a hack than something really supported I think this is not necessary.

Reply:
instead of boolean replaced with 'Y'